packages/util/src/environment.ts

```diff
@@ -83,6 +83,16 @@ export function isReactNative(): boolean {
   );
 }
 
+/**
+ * Detect Electron.
+ *
+ * @return true if Electron environment is detected.
+ */
+export function isElectron(): boolean {
+  const userAgent = navigator.userAgent.toLowerCase();
```

Comment:
`navigator` may not be defined in some environments (e.g. node.js) and so this could crash.  The easy solution is to use the existing `getUA()` helper.